### PR TITLE
Recipe Details screen

### PR DESCRIPTION
**Pull request by MaximeRougieux**
Trello ticket : https://trello.com/c/nJUDRCET/57-etqnb-quand-je-clique-sur-un-item-de-la-liste-je-navigue-vers-la-page-de-d%C3%A9tails-de-cet-item-navigation

Capture d’écran 2022-05-06 à 18 02 17

### CODE REVIEW

File: src/recipes/screens/RecipeDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+interface RecipeInstructionProps {
+  item: string;
+  index: number;
+}
+
+const RecipeInstruction = ({ item, index }: RecipeInstructionProps) => {
+  return (
+    <View style={styles.instructionContainer}>
+      <Text>{`${index}.   `}</Text>
```
Comment (T-moz) [inline src/recipes/screens/RecipeDetails.tsx]: Why is there a ` ` in your text ?

File: src/recipes/screens/RecipeDetails.tsx [hunk 2]
```diff
@@ -20,16 +17,23 @@ const RecipeInstruction = ({ item, index }: RecipeInstructionProps) => {
   );
 };
 
-export const RecipeDetails = ({ meal }: RecipeDetailsProps) => {
+const getInstructionsComponents = (meal: Meal) =>
```
Comment (T-moz) [inline src/recipes/screens/RecipeDetails.tsx:20]: Creat a React component

File: src/recipes/screens/RecipeList.tsx [hunk 3]
```diff
@@ -20,22 +29,32 @@ export const RecipeList = () => {
...
         ) : (
           items.map(meal => (
-            <RecipeItem
-              name={meal.strMeal}
-              category={meal.strCategory}
-              imageUrl={meal.strMealThumb}
+            <TouchableOpacity
               key={meal.idMeal}
-            />
+              onPress={() => navigation.navigate('Details', { meal: meal })}>
```
Comment (T-moz) [inline src/recipes/screens/RecipeList.tsx:34]: Put this function in a `useCallback`
https://fr.reactjs.org/docs/hooks-reference.html#usecallback

**Comment by T-moz** [comment]
It's cool that you created more commits.
But keep in mind that every commit should be "complete". This means tests and lints work at every commit.

**Comment by T-moz** [comment]
We can fix commit history together if you want

**Comment by MaximeRougieux** [comment]
> We can fix commit history together if you want

Sure, I'm going to fix the issues you've raised, and then we can do this together (after lunch maybe if that's ok for you).

**Pull request merged**